zproject/backends.py

```diff
@@ -1027,6 +1029,9 @@ def sync_groups_from_ldap(self, user_profile: UserProfile, ldap_user: _LDAPUser)
                 intended_group_names=intended_group_names,
                 user_profile=user_profile,
                 logger=ldap_logger,
+                # TODO: should we make this True? False is the backward compat way.
+                # But True seems like the actual behavior that makes much more sense.
+                create_missing_groups=False,
```

Comment:
For 12.0, we set it to True.  If we're backporting to 11.x, we leave it False, I expect.